website/docs/r/user_schema.md

```diff
@@ -252,7 +252,7 @@ The following arguments are supported:
 
 In addition to the above arguments, the following attributes are exported:
 
-* `schema_id`
+* `schema_id` - Unique identifier of Schema.
```

Comment:
The above was copied from the URL:
https://github.com/googleapis/google-api-go-client/blob/master/admin/directory/v1/admin-gen.go#L3320